### ISSUE TITLE
PaddingIdx

### ISSUE THREAD

**Issue by MogicianXD** [rixwew/pytorch-fm#23]
Is it necessary to set the arg `padding_idx` of `torch.nn.embbedding`, as all features start from 1.
I think it's better to avoid influence of 0.

**Comment by MogicianXD** [comment]
I have found it is important to set padding idx.

Take it into considerations?

```python
class FeaturesEmbedding(torch.nn.Module):

    def __init__(self, field_dims, embed_dim):
        super().__init__()
        self.embedding = torch.nn.Embedding(sum(field_dims), embed_dim, padding_idx=0)
        self.offsets = np.array((0, *np.cumsum(field_dims)[:-1]), dtype=np.long)
        torch.nn.init.xavier_uniform_(self.embedding.weight.data)

    def forward(self, x):
        """
        :param x: Long tensor of size ``(batch_size, num_fields)``
        """
        mask = x > 0
        x = x + x.new_tensor(self.offsets).unsqueeze(0)
        x = torch.where(mask, x.new_zeros(x.shape), x)
        return self.embedding(x)
```

**Comment by rixwew** [comment]
Thank you for the suggestion!
Features starting from zero may not be in use now,  but I sometime use them for private projects.
I'd like to keep it that way because It doesn't affect the accuracy of the models.
